app/controllers/articles_controller.rb

```diff
@@ -13,7 +13,7 @@ def new
   end
 
   def create
-    @article = Article.new(article_params) # Article.new(title: 'title', body: 'body')
+    @article = current_user.articles.build(article_params)
```

Comment:
normally we use `.new` here
```suggestion
    @article = current_user.articles.new(article_params)
```